### ISSUE TITLE
@cdn incorrectly configuring Origin Path with new http api gateway

### ISSUE THREAD

**Issue by anatomic** [architect/architect#965]
**Describe the issue**
Deploying using `arc deploy` when `app.arc` contains the `@cdn` pragma sets the origin for the API Cloudfront distribution to `s9m2o9qxx5.execute-api.eu-west-2.amazonaws.com/<<stage>>` but the HTTP Api Gateway deployment only includes a `$default` stage.

**Steps to reproduce**
1. `arc init`
2. Add static assets to `/public`
3. add `@cdn` and `@static` to `app.arc`
4. `arc deploy`
5. Inspect Cloudfront distribution created to front the API Gateway
6. notice that `Origin Path` includes the stage you just deployed to

**Expected behavior**
Deploying with architect 7 using the default HTTP API Gateway does not set `Origin Path` to `/<<stage>>`

**Screenshots**
![image](https://user-images.githubusercontent.com/152841/93830004-e9f17280-fc66-11ea-936d-f7e7be1213c7.png)

![image](https://user-images.githubusercontent.com/152841/93830266-a4817500-fc67-11ea-94ec-623ad9b7dfc2.png)

**Desktop**
N/A

**Mobile**
N/A

**Additional context**

Critical part of `app.arc`
```
@cdn

@static
fingerprint true

@http
get /
get /login
get /auth
get /logout
get /activities
get /activities/:page
post /api/update-equipment
```

```
➜ arc version
      Version: Architect 7.0.2
 Installed to: /Users/ith02/.asdf/installs/nodejs/13.13.0/.npm/lib/node_modules/@architect/architect
          cwd: /Users/ith02/dev/github/anatomic/strava-bulk-edit-arc
```

**Comment by anatomic** [comment]
Manually editing the cloudfront distribution to remove the path results in static assets appearing as expected

**Comment by ryanblock** [comment]
Patched in 7.0.3 RC, awaiting verification from someone that is not me!

**Comment by anatomic** [comment]
I created a new arc project and deploy staging with 7.0.2 and production with 7.0.3-RC.0 and it looks like this fixes the issue! Great work Ryan

![image](https://user-images.githubusercontent.com/152841/94236613-70c37b00-ff05-11ea-9225-9220e81de4b0.png)
